cloud.html

```diff
@@ -2,97 +2,23 @@
 <html>
 
 <head>
-  <title>xrpackage | trees</title>
-  <link rel=icon type="image/png" href="favicon.png">
-  <link href="/index.css" rel=stylesheet>
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
-  <script src="https://kit.fontawesome.com/0735724151.js" crossorigin="anonymous"></script>
+  <title>cloud</title>
 </head>
 
-<body class="header-gap">
-  <header class=header id=header>
-    <a href="/" class="nav icon">山</a>
-    <nav class="selector nav bar" id=selector>
-      <i class="fa fa-user head"></i>
-      <div class=header-label id=header-label>Log in</div>
-      <i class="fa fa-chevron-down tail"></i>
-      <div class=selections id=selections>
-        <a href="run.html" class=selection id=run-mode>
-          <i class="fa fa-play"></i>
-          <div class=wrap>
-            <h3>Run mode</h3>
-            <p>Switch to run mode</p>
-          </div>
-        </a>
-        <a href="edit.html" class=selection id=edit-mode>
-          <i class="fa fa-pencil"></i>
-          <div class=wrap>
-            <h3>Edit mode</h3>
-            <p>You are currently editing this world</p>
-          </div>
-        </a>
-        <a href="browse.html" class="selection dim">
-          <i class="fa fa-solar-system"></i>
-          <div class=wrap>
-            <h3>Browse packages</h3>
-            <p>Explore the XRPackage registry</p>
-          </div>
-        </a>
-        <a href="help.html" class=selection>
-          <i class="fa fa-book"></i>
-          <div class=wrap>
-            <h3>Documentation</h3>
-            <p>Learn how to make packages and worlds</p>
-          </div>
-        </a>
-      </div>
-    </nav>
-
-    <form class="login-form phase-1" id=login-form>
-      <div class=phase-content>
-        <div class=login-notice id=login-notice></div>
-        <div class=login-error id=login-error></div>
-      </div>
-      <div class="phase-content phase-1-content">
-        <input type=email placeholder="your@email.com" id=login-email>
-        <input type=submit value="Log in" class="button highlight">
-      </div>
-      <div class="phase-content phase-2-content">
-        <input type=text placeholder="Verification code" id=login-verification-code>
-        <input type=submit value="Verify" class="button highlight">
-      </div>
-      <div class="phase-content phase-3-content">
-        <nav class=user-button id=user-button>
-          <img src="favicon.ico">
-          <span class=name id=login-email-static></span>
-          <input type=submit value="Log out" class="button highlight">
-        </nav>
-      </div>
-      <div class="phase-content phaseless-content">
-        <div>Working...</div>
-      </div>
-    </form>
-  </header>
+<body>
 
 <script type=module>
-
-import * as THREE from './three.module.js';
-import {OrbitControls} from './OrbitControls.js';
+import * as THREE from 'https://static.xrpackage.org/xrpackage/three.module.js';
```

Comment:
The only thing that should refer to these internal URLs is possibly tightly knot things like the site -- not third party packages.

In the future we might use Import Maps to coalesce library loads like THREE.js but, but until then packages should probably ship their own copy of THREE.js.

Reply:
I see, that makes sense! I was originally thinking this would help stuff like caching if all the packages were just getting the js from the remote but it looks like Import Maps would solve this anyway!

Should we minify the `three.module.js` files we use then? That itself is 1161kB, which seems fairly large to bundle with each package to me.

Reply:
I think uglifying is fine, though I'm not sure what level of savings that will give us.

Reply:
I used https://www.npmjs.com/package/terser to minify it and it's gone from 1161kB to 610kB now!